resources/prosody-plugins/mod_muc_lobby_rooms.lua

```diff
@@ -507,6 +507,24 @@ process_host_module(main_muc_component_config, function(host_module, host)
             end
         end
     end);
+
+    -- listen for admin set
+    for event_name, method in pairs {
+        -- Normal room interactions
+        ["iq-set/bare/http://jabber.org/protocol/muc#admin:query"] = "handle_admin_query_set_command" ;
+        -- Host room
+        ["iq-set/host/http://jabber.org/protocol/muc#admin:query"] = "handle_admin_query_set_command" ;
+    } do
+        host_module:hook(event_name, function (event)
+            local origin, stanza = event.origin, event.stanza;
+            local room_jid = jid_bare(stanza.attr.to);
+            local room = get_room_from_jid(room_jid);
+
+            if room then
+                return handle_admin_query_set_command(room, origin, stanza);
```

Comment:
Do we want some a log and possibly an error response if there's no `room`?

Reply:
I can add one, but we currently don't have one... as this is again copy from their code. 
https://github.com/bjc/prosody/blob/2798f91821c0e98ce8ed7f33b4c776289ad1d7f8/plugins/muc/mod_muc.lua#L464